avm/res/kubernetes-configuration/extension/main.bicep

```diff
@@ -94,7 +94,7 @@ module fluxConfiguration 'br/public:avm/res/kubernetes-configuration/flux-config
     bucket: fluxConfiguration.?bucket
     configurationProtectedSettings: fluxConfiguration.?configurationProtectedSettings
     gitRepository: fluxConfiguration.?gitRepository
-    kustomizations: fluxConfiguration.?kustomizations
+    kustomizations: fluxConfiguration.kustomizations
```

Comment:
I think we should avoid releasing kube extensions in this PR. I'd suggest to:
Update kube config and release 0.3.0 in this PR
Update kube extensions in PR #613

Reply:
Done @eriqua